bool                              acknowledged
mapc_rhbp_ettlinger/AssembleBid     bid
